app/api/api_v1/endpoints/assignment_router.py

```diff
@@ -47,8 +47,6 @@ async def update_assignment_fields(
     update_schema = UpdateAssignmentSchema(**updated_set_fields)
     assignment = await AssignmentService(db).update_assignment(assignment, update_schema)
 
-    await LmsSyncService(db).upsync_assignment(assignment)
```

Comment:
why are we deleting the upsync here? dont we use this endpoint to update the assignment dates back to canvas?

Reply:
ah yes I see its in the service now, ignore this